.github/workflows/tests.yml

```diff
@@ -0,0 +1,91 @@
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
+name: Run tests
+
+jobs:
+
+  precommit:
+    name: Run the precommit checks
```

Comment:
Maybe call it "Pre-commit checks", to be consistent with the rest?

Reply:
updated. also updated the mergify commit to reflect this change